.vsts.pipelines/jobs/ci-local.yml

```diff
@@ -68,19 +72,23 @@ jobs:
   - template: ../steps/check-space-powershell.yml
 
   # Build source-build.
-  - script: ${{ format('{0}build{1} $(args.build)', parameters.scriptPrefix, parameters.scriptSuffix) }}
-    env:
-      internalPackageFeedPat: $(InternalPackageFeedPatEnv)
+  - script: |
+      set -x
+      $(SetInternalPackageFeedPatBashCommand)
```

Comment:
Does `set -e` cause a failure for an empty command?  Otherwise could do our standard `exuo pipefail` here.

Reply:
`$()` exits 0, but we don't actually have an empty command here--we basically have `x=$(asdf)`. `asdf` can't be found, so the subshell command exits nonzero.